build/linux/installer/conf/kube-cm.conf

```diff
@@ -0,0 +1,358 @@
+   #fluent forward plugin
+   <system>
+    workers "#{ENV['NUM_OF_FLUENTD_WORKERS']}"
+    root_dir /var/opt/microsoft/docker-cimprov/state
+   </system>
+
+   #@include windows_rs_containerinventory.conf
```

Comment:
Why is this in the custom metrics kube file? It should be in the previous file itself

Reply:
this file's config is what we have now. the file kube.conf is the one without custom metrics. 

Are you saying that this should be present in both files?

Reply:
I just checked and this line is present in both files. @jatakiajanvi12